requirements.txt

```diff
@@ -76,7 +76,8 @@ docopt==0.6.2
     # via notifications-python-client
 docutils==0.15.2
     # via awscli
-eventlet==0.30.2
+# pip-compile will remove this pyup comment, make sure the comment is added back (unless gunicorn is bumped >20.1.0).
+eventlet==0.30.2 # pyup: ignore
```

Comment:
I don't mind this being tacked on, but I'm not sure I agree with it: while we may have no intention of upgrading it soon, there's still a security issue with it and silencing that feels dodgy to me, since we forget it's there. Should we just let it be?

Reply:
Coming back to old PRs. i've changed my view a bit, i'm with you, lets remove this pyup directive.